backend/cats/src/app/utilities/enums/application/sortByDirection.enum.ts

```diff
@@ -0,0 +1,10 @@
+import { registerEnumType } from '@nestjs/graphql';
+
+export enum SortByDirection {
+  ASC = 'ASC',
+  DESC = 'DESC',
+}
```

Comment:
my only suggestion here would be to have this enum as a common one instead of placing it under application folder - in case sort by direction can be used in other tables